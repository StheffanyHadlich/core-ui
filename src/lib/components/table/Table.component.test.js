import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/table/__snapshots__"
});
