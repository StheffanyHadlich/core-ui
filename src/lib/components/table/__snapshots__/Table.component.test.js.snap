// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Table Default 1`] = `
<div
  style={
    Object {
      "height": "100vh",
    }
  }
>
  <div
    className="sc-table"
    style={
      Object {
        "height": 0,
        "overflow": "visible",
        "width": 0,
      }
    }
  >
    <div
      className="sc-bdVaJa iyEjyO"
    >
      <div
        aria-colcount={6}
        aria-rowcount={100}
        className="ReactVirtualized__Table"
        role="grid"
        style={Object {}}
      >
        <div
          className="ReactVirtualized__Table__headerRow sc-table-row"
          role="row"
          style={
            Object {
              "height": 40,
              "overflow": "hidden",
              "paddingRight": 0,
              "width": 0,
            }
          }
        >
          <div
            aria-label="Id"
            aria-sort="none"
            className="ReactVirtualized__Table__headerColumn sc-table-header ReactVirtualized__Table__sortableHeaderColumn"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="columnheader"
            style={
              Object {
                "WebkitFlex": "1 1 200px",
                "flex": "1 1 200px",
                "msFlex": "1 1 200px",
              }
            }
            tabIndex={0}
          >
            <div
              className="sc-bwzfXH jIEWCi"
            >
              <label>
                Id
              </label>
              <div
                className="sc-htpNat dkojKu"
                selected={false}
              >
                <i
                  className="fas fa-sort-up"
                />
                <i
                  className="fas fa-sort-down"
                />
              </div>
            </div>
          </div>
          <div
            aria-label="First Name"
            aria-sort="ascending"
            className="ReactVirtualized__Table__headerColumn sc-table-header ReactVirtualized__Table__sortableHeaderColumn"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="columnheader"
            style={
              Object {
                "WebkitFlex": "1 1 200px",
                "flex": "1 1 200px",
                "msFlex": "1 1 200px",
              }
            }
            tabIndex={0}
          >
            <div
              className="sc-bwzfXH jIEWCi"
            >
              <label>
                First Name
              </label>
              <div
                className="sc-htpNat iHhtxD"
                selected={true}
              >
                <i
                  className="fas fa-sort-up"
                />
                <i
                  className="fas fa-sort-down"
                />
              </div>
            </div>
          </div>
          <div
            aria-label="Last Name"
            aria-sort="none"
            className="ReactVirtualized__Table__headerColumn sc-table-header ReactVirtualized__Table__sortableHeaderColumn"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="columnheader"
            style={
              Object {
                "WebkitFlex": "1 1 200px",
                "flex": "1 1 200px",
                "msFlex": "1 1 200px",
              }
            }
            tabIndex={0}
          >
            <div
              className="sc-bwzfXH jIEWCi"
            >
              <label>
                Last Name
              </label>
              <div
                className="sc-htpNat dkojKu"
                selected={false}
              >
                <i
                  className="fas fa-sort-up"
                />
                <i
                  className="fas fa-sort-down"
                />
              </div>
            </div>
          </div>
          <div
            className="ReactVirtualized__Table__headerColumn sc-table-header"
            role="columnheader"
            style={
              Object {
                "WebkitFlex": "1 1 200px",
                "flex": "1 1 200px",
                "msFlex": "1 1 200px",
              }
            }
          >
            <div
              className="sc-bwzfXH jIEWCi"
            >
              <label>
                Email
              </label>
            </div>
          </div>
          <div
            className="ReactVirtualized__Table__headerColumn sc-table-header"
            role="columnheader"
            style={
              Object {
                "WebkitFlex": "1 1 200px",
                "flex": "1 1 200px",
                "msFlex": "1 1 200px",
              }
            }
          >
            <div
              className="sc-bwzfXH jIEWCi"
            >
              <label>
                Ip Address
              </label>
            </div>
          </div>
          <div
            aria-label="Age"
            aria-sort="none"
            className="ReactVirtualized__Table__headerColumn sc-table-header ReactVirtualized__Table__sortableHeaderColumn"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="columnheader"
            style={
              Object {
                "WebkitFlex": "1 1 200px",
                "flex": "1 1 200px",
                "msFlex": "1 1 200px",
              }
            }
            tabIndex={0}
          >
            <div
              className="sc-bwzfXH jIEWCi"
            >
              <label>
                Age
              </label>
              <div
                className="sc-htpNat dkojKu"
                selected={false}
              >
                <i
                  className="fas fa-sort-up"
                />
                <i
                  className="fas fa-sort-down"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          aria-label="grid"
          aria-readonly={true}
          className="ReactVirtualized__Grid ReactVirtualized__Table__Grid"
          onScroll={[Function]}
          role="rowgroup"
          style={
            Object {
              "WebkitOverflowScrolling": "touch",
              "boxSizing": "border-box",
              "direction": "ltr",
              "height": -40,
              "overflowX": "hidden",
              "overflowY": "auto",
              "position": "relative",
              "width": 0,
              "willChange": "transform",
            }
          }
          tabIndex={0}
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Table Empty Table 1`] = `
<div
  style={
    Object {
      "height": "100vh",
    }
  }
>
  <div
    className="sc-table"
    style={
      Object {
        "height": 0,
        "overflow": "visible",
        "width": 0,
      }
    }
  >
    <div
      className="sc-bdVaJa iyEjyO"
    >
      <div
        aria-colcount={6}
        aria-rowcount={0}
        className="ReactVirtualized__Table"
        role="grid"
        style={Object {}}
      >
        <div
          className="ReactVirtualized__Table__headerRow sc-table-row"
          role="row"
          style={
            Object {
              "height": 40,
              "overflow": "hidden",
              "paddingRight": 0,
              "width": 0,
            }
          }
        >
          <div
            aria-label="Id"
            aria-sort="none"
            className="ReactVirtualized__Table__headerColumn sc-table-header ReactVirtualized__Table__sortableHeaderColumn"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="columnheader"
            style={
              Object {
                "WebkitFlex": "1 1 200px",
                "flex": "1 1 200px",
                "msFlex": "1 1 200px",
              }
            }
            tabIndex={0}
          >
            <div
              className="sc-bwzfXH jIEWCi"
            >
              <label>
                Id
              </label>
              <div
                className="sc-htpNat dkojKu"
                selected={false}
              >
                <i
                  className="fas fa-sort-up"
                />
                <i
                  className="fas fa-sort-down"
                />
              </div>
            </div>
          </div>
          <div
            aria-label="First Name"
            aria-sort="none"
            className="ReactVirtualized__Table__headerColumn sc-table-header ReactVirtualized__Table__sortableHeaderColumn"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="columnheader"
            style={
              Object {
                "WebkitFlex": "1 1 200px",
                "flex": "1 1 200px",
                "msFlex": "1 1 200px",
              }
            }
            tabIndex={0}
          >
            <div
              className="sc-bwzfXH jIEWCi"
            >
              <label>
                First Name
              </label>
              <div
                className="sc-htpNat dkojKu"
                selected={false}
              >
                <i
                  className="fas fa-sort-up"
                />
                <i
                  className="fas fa-sort-down"
                />
              </div>
            </div>
          </div>
          <div
            aria-label="Last Name"
            aria-sort="descending"
            className="ReactVirtualized__Table__headerColumn sc-table-header ReactVirtualized__Table__sortableHeaderColumn"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="columnheader"
            style={
              Object {
                "WebkitFlex": "1 1 200px",
                "flex": "1 1 200px",
                "msFlex": "1 1 200px",
              }
            }
            tabIndex={0}
          >
            <div
              className="sc-bwzfXH jIEWCi"
            >
              <label>
                Last Name
              </label>
              <div
                className="sc-htpNat hrnmMY"
                selected={true}
              >
                <i
                  className="fas fa-sort-up"
                />
                <i
                  className="fas fa-sort-down"
                />
              </div>
            </div>
          </div>
          <div
            className="ReactVirtualized__Table__headerColumn sc-table-header"
            role="columnheader"
            style={
              Object {
                "WebkitFlex": "1 1 200px",
                "flex": "1 1 200px",
                "msFlex": "1 1 200px",
              }
            }
          >
            <div
              className="sc-bwzfXH jIEWCi"
            >
              <label>
                Email
              </label>
            </div>
          </div>
          <div
            className="ReactVirtualized__Table__headerColumn sc-table-header"
            role="columnheader"
            style={
              Object {
                "WebkitFlex": "1 1 200px",
                "flex": "1 1 200px",
                "msFlex": "1 1 200px",
              }
            }
          >
            <div
              className="sc-bwzfXH jIEWCi"
            >
              <label>
                Ip Address
              </label>
            </div>
          </div>
          <div
            aria-label="Age"
            aria-sort="none"
            className="ReactVirtualized__Table__headerColumn sc-table-header ReactVirtualized__Table__sortableHeaderColumn"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="columnheader"
            style={
              Object {
                "WebkitFlex": "1 1 200px",
                "flex": "1 1 200px",
                "msFlex": "1 1 200px",
              }
            }
            tabIndex={0}
          >
            <div
              className="sc-bwzfXH jIEWCi"
            >
              <label>
                Age
              </label>
              <div
                className="sc-htpNat dkojKu"
                selected={false}
              >
                <i
                  className="fas fa-sort-up"
                />
                <i
                  className="fas fa-sort-down"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          aria-label="grid"
          aria-readonly={true}
          className="ReactVirtualized__Grid ReactVirtualized__Table__Grid"
          onScroll={[Function]}
          role="rowgroup"
          style={
            Object {
              "WebkitOverflowScrolling": "touch",
              "boxSizing": "border-box",
              "direction": "ltr",
              "height": -40,
              "overflowX": "hidden",
              "overflowY": "auto",
              "position": "relative",
              "width": 0,
              "willChange": "transform",
            }
          }
          tabIndex={0}
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Table ThemeProvider 1`] = `
<div>
  <div
    style={
      Object {
        "height": "100vh",
      }
    }
  >
    <div
      className="sc-table"
      style={
        Object {
          "height": 0,
          "overflow": "visible",
          "width": 0,
        }
      }
    >
      <div
        className="sc-bdVaJa dTciQw"
      >
        <div
          aria-colcount={6}
          aria-rowcount={100}
          className="ReactVirtualized__Table"
          role="grid"
          style={Object {}}
        >
          <div
            className="ReactVirtualized__Table__headerRow sc-table-row"
            role="row"
            style={
              Object {
                "height": 40,
                "overflow": "hidden",
                "paddingRight": 0,
                "width": 0,
              }
            }
          >
            <div
              aria-label="Id"
              aria-sort="none"
              className="ReactVirtualized__Table__headerColumn sc-table-header ReactVirtualized__Table__sortableHeaderColumn"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="columnheader"
              style={
                Object {
                  "WebkitFlex": "1 1 200px",
                  "flex": "1 1 200px",
                  "msFlex": "1 1 200px",
                }
              }
              tabIndex={0}
            >
              <div
                className="sc-bwzfXH jIEWCi"
              >
                <label>
                  Id
                </label>
                <div
                  className="sc-htpNat dkojKu"
                  selected={false}
                >
                  <i
                    className="fas fa-sort-up"
                  />
                  <i
                    className="fas fa-sort-down"
                  />
                </div>
              </div>
            </div>
            <div
              aria-label="First Name"
              aria-sort="ascending"
              className="ReactVirtualized__Table__headerColumn sc-table-header ReactVirtualized__Table__sortableHeaderColumn"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="columnheader"
              style={
                Object {
                  "WebkitFlex": "1 1 200px",
                  "flex": "1 1 200px",
                  "msFlex": "1 1 200px",
                }
              }
              tabIndex={0}
            >
              <div
                className="sc-bwzfXH jIEWCi"
              >
                <label>
                  First Name
                </label>
                <div
                  className="sc-htpNat kgsZKr"
                  selected={true}
                >
                  <i
                    className="fas fa-sort-up"
                  />
                  <i
                    className="fas fa-sort-down"
                  />
                </div>
              </div>
            </div>
            <div
              aria-label="Last Name"
              aria-sort="none"
              className="ReactVirtualized__Table__headerColumn sc-table-header ReactVirtualized__Table__sortableHeaderColumn"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="columnheader"
              style={
                Object {
                  "WebkitFlex": "1 1 200px",
                  "flex": "1 1 200px",
                  "msFlex": "1 1 200px",
                }
              }
              tabIndex={0}
            >
              <div
                className="sc-bwzfXH jIEWCi"
              >
                <label>
                  Last Name
                </label>
                <div
                  className="sc-htpNat dkojKu"
                  selected={false}
                >
                  <i
                    className="fas fa-sort-up"
                  />
                  <i
                    className="fas fa-sort-down"
                  />
                </div>
              </div>
            </div>
            <div
              className="ReactVirtualized__Table__headerColumn sc-table-header"
              role="columnheader"
              style={
                Object {
                  "WebkitFlex": "1 1 200px",
                  "flex": "1 1 200px",
                  "msFlex": "1 1 200px",
                }
              }
            >
              <div
                className="sc-bwzfXH jIEWCi"
              >
                <label>
                  Email
                </label>
              </div>
            </div>
            <div
              className="ReactVirtualized__Table__headerColumn sc-table-header"
              role="columnheader"
              style={
                Object {
                  "WebkitFlex": "1 1 200px",
                  "flex": "1 1 200px",
                  "msFlex": "1 1 200px",
                }
              }
            >
              <div
                className="sc-bwzfXH jIEWCi"
              >
                <label>
                  Ip Address
                </label>
              </div>
            </div>
            <div
              aria-label="Age"
              aria-sort="none"
              className="ReactVirtualized__Table__headerColumn sc-table-header ReactVirtualized__Table__sortableHeaderColumn"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="columnheader"
              style={
                Object {
                  "WebkitFlex": "1 1 200px",
                  "flex": "1 1 200px",
                  "msFlex": "1 1 200px",
                }
              }
              tabIndex={0}
            >
              <div
                className="sc-bwzfXH jIEWCi"
              >
                <label>
                  Age
                </label>
                <div
                  className="sc-htpNat dkojKu"
                  selected={false}
                >
                  <i
                    className="fas fa-sort-up"
                  />
                  <i
                    className="fas fa-sort-down"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            aria-label="grid"
            aria-readonly={true}
            className="ReactVirtualized__Grid ReactVirtualized__Table__Grid"
            onScroll={[Function]}
            role="rowgroup"
            style={
              Object {
                "WebkitOverflowScrolling": "touch",
                "boxSizing": "border-box",
                "direction": "ltr",
                "height": -40,
                "overflowX": "hidden",
                "overflowY": "auto",
                "position": "relative",
                "width": 0,
                "willChange": "transform",
              }
            }
            tabIndex={0}
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
